pephub/routers/api/v1/project.py

```diff
@@ -30,11 +31,20 @@
 async def get_a_pep(
     proj: peppy.Project = Depends(get_project),
     proj_annotation: AnnotationModel = Depends(get_project_annotation),
+    raw: bool = False,
 ):
     """
     Fetch a PEP from a certain namespace
     """
-
+    if raw:
+        raw_project = proj.to_dict(extended=True)
+        try:
+            raw_project = ProjectRawModel(**raw_project)
+        except Exception as err:
```

Comment:
I'd leave out this `try: catch:` logic, and let the server just fail and thow the `500` itself. [See here](https://sqa.stackexchange.com/a/2990) for why its more secure to not propagate the stack trace out to the user.

Reply:
hmm, it's a good point!

Reply:
After discussion, we ended up that we don't want to change peppy.
So issue #124 is blocking merging this PR :/